README.md

```diff
@@ -16,11 +16,13 @@ Variable | Description
 ---------|------------
 project_name | Human readable name of your microservice
 project_slug | Machine-friendly version of project_name (directories/code/etc. will use this)
+project_upper | Upper case version of the project slug
 app_name | Importable name of your app (should differ from project_slug)
 dc_port | Port that will appear in the docker-compose.yml for the local environment
-use_sqlalchemy | True/False whether to include SQLAlchemy modules/code for MariaDB
+use_sqlalchemy | y/n whether to include SQLAlchemy modules/code for MariaDB
```

Comment:
Why switch to y/n? Just want less typing?

Reply:
Yeah, and if you type in true/false vs True/False you get an error in python.